### PR TITLE
Feature/storybook

### PR DESCRIPTION
**Pull request by Legitgoons**
## Description
- Set up `Storybook` and automated deployment
- Updated `.npmignore` & `.gitignore`
## Changes
- Installed Storybook.
- Implemented `publish.yml` using GitHub Actions for automated Storybook deployment.
  - Additional configuration is required because the implementation is set to deploy to GitHub Pages. For more details, please refer to [this guide](https://github.com/marketplace/actions/deploy-storybook-to-github-pages#set-up-your-project-for-actions-deployed-pages).

### CODE REVIEW

**Review by BangDori — COMMENTED** [review]
node 버전만 추가해주시고, storybook 기본 이미지 파일들은 모두 제거해주셔도 될 것 같습니다~

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
```
[the remark this answers is not in the document]
Comment (Legitgoons) [reply .github/workflows/publish.yml:19]: 넵 추가하겠습니다

**Review by BangDori — APPROVED** [review]
확인했습니당~ workflow 실행하셔서 테스트 이후에 병합 진행해주세요!

**Comment by Legitgoons** [comment]
package의 사이즈를 고려해 storybook용 repo를 따로 만들겠습니다!

**Pull request closed without merging**